k8s/terraform/main.tf

```diff
@@ -15,4 +15,5 @@ resource "juju_application" "vault-k8s" {
   config      = var.config
   constraints = var.constraints
   units       = var.units
+  trust       = var.trust
```

Comment:
Does it make sense to make this configurable? Do we properly handle everything if trust is not granted?